runtime/tests/checker/entitlements_test.go

```diff
@@ -1535,52 +1509,19 @@ func TestCheckBasicEntitlementMappingAccess(t *testing.T) {
 		assert.NoError(t, err)
 	})
 
-	t.Run("multiple mappings conjunction", func(t *testing.T) {
+	t.Run("invalid mapping", func(t *testing.T) {
 		t.Parallel()
 
 		_, err := ParseAndCheck(t, `
-            entitlement mapping M {}
-            entitlement mapping N {}
-            resource interface R {
-                access(M, N) let foo: String
-            }
-        `)
-
-		errs := RequireCheckerErrors(t, err, 1)
-
-		require.IsType(t, &sema.InvalidMultipleMappedEntitlementError{}, errs[0])
-	})
-
-	t.Run("multiple mappings conjunction with regular", func(t *testing.T) {
-		t.Parallel()
-
-		_, err := ParseAndCheck(t, `
-            entitlement mapping M {}
-            entitlement N
-            resource interface R {
-                access(M, N) let foo: String
-            }
-        `)
-
-		errs := RequireCheckerErrors(t, err, 1)
-
-		require.IsType(t, &sema.InvalidMultipleMappedEntitlementError{}, errs[0])
-	})
-
-	t.Run("multiple mappings disjunction", func(t *testing.T) {
-		t.Parallel()
-
-		_, err := ParseAndCheck(t, `
-            entitlement mapping M {}
-            entitlement mapping N {}
+            entitlement M
             resource interface R {
-                access(M | N) let foo: String
+                access(mapping M) let foo: String
```

Comment:
Is there a test for the opposite: Using an entitlement mapping, without the `mapping` keyword?

Reply:
Added